src/iqm/qiskit_iqm/iqm_move_layout.py

```diff
@@ -139,8 +139,8 @@ def _require_type(qubit_index: int, required_type: str, instruction_name: str):
                 # The move operation requires that the first operand is the move qubit,
                 # and the second must be the resonator
                 (qubit, resonator) = node.qargs
-                _require_type(qubit.index, 'move_qubit', 'move')
-                _require_type(resonator.index, 'resonator', 'move')
+                _require_type(qubit._index, 'move_qubit', 'move')
+                _require_type(resonator._index, 'resonator', 'move')
```

Comment:
Maybe we should not use private attributes. Is there some other, recommended way of doing this?

Reply:
Elsewhere as well.

Reply:
Try to find these instances and remove them if trivial. In cases where this cannot be simply fixed, I'll make a note and discussion point.

Reply:
I managed to remove all instances. In some cases, this might cause a small performance hit for looking up the index of a qubit from the circuit, but all those cases can be removed in the near future as part of the traspiler improvements ticket COMP-1361.